diff --git a/Makefile b/Makefile
index bfdc92c..39f6a09 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,8 @@
 VERSION = 4
 PATCHLEVEL = 12
 SUBLEVEL = 4
-EXTRAVERSION =
-NAME = Fearless Coyote
+EXTRAVERSION = .syscall
+NAME = Rathod Raja V
 
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
diff --git a/arch/x86/entry/syscalls/syscall_64.tbl b/arch/x86/entry/syscalls/syscall_64.tbl
index dca8eef..149b21d 100644
--- a/arch/x86/entry/syscalls/syscall_64.tbl
+++ b/arch/x86/entry/syscalls/syscall_64.tbl
@@ -339,6 +339,7 @@
 330	common	pkey_alloc		sys_pkey_alloc
 331	common	pkey_free		sys_pkey_free
 332	common	statx			sys_statx
+333     64      myhello                 sys_myhello
 #
 # x32-specific system call numbers start at 512 to avoid cache impact
 # for native 64-bit operation.
diff --git a/include/linux/syscalls.h b/include/linux/syscalls.h
index 37414d1..4cb9035 100644
--- a/include/linux/syscalls.h
+++ b/include/linux/syscalls.h
@@ -908,3 +908,4 @@ asmlinkage long sys_statx(int dfd, const char __user *path, unsigned flags,
 
 #endif
 
+asmlinkage int sys_myhello(void);
diff --git a/kernel/Makefile b/kernel/Makefile
index 72aa080..d81bc88 100644
--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -5,7 +5,7 @@
 obj-y     = fork.o exec_domain.o panic.o \
 	    cpu.o exit.o softirq.o resource.o \
 	    sysctl.o sysctl_binary.o capability.o ptrace.o user.o \
-	    signal.o sys.o kmod.o workqueue.o pid.o task_work.o \
+	    signal.o sys.o myhello.o kmod.o workqueue.o pid.o task_work.o \
 	    extable.o params.o \
 	    kthread.o sys_ni.o nsproxy.o \
 	    notifier.o ksysfs.o cred.o reboot.o \
